code/modules/power/reactor/rbmk.dm

```diff
@@ -4,28 +4,85 @@
 #define MODERATOR_INPUT_GATE airs[2]
 #define COOLANT_OUTPUT_GATE airs[3]
 
-#define RBMK_TEMPERATURE_OPERATING 640 //Celsius
-#define RBMK_TEMPERATURE_CRITICAL 810 //At this point the entire ship/station is alerted to a meltdown. This may need altering
-#define RBMK_TEMPERATURE_MELTDOWN 900
+#define RBMK_TEMPERATURE_OPERATING 640+T0C
+#define RBMK_TEMPERATURE_CRITICAL 810+T0C //At this point the entire ship/station is alerted to a meltdown. This may need altering
+#define RBMK_TEMPERATURE_MELTDOWN 900+T0C
```

Comment:
i know this is like a super old PR that i am just now getting to look at but. shouldn't this be _minus_ T0C because the numbers were originally presented in celsius?